pkg/project/version_bundle.go

```diff
@@ -68,6 +68,6 @@ func VersionBundle(p string) versionbundle.Bundle {
 		},
 		Name:     "cluster-operator",
 		Provider: p,
-		Version:  "2.0.0-dev",
+		Version:  BundleVersion(),
```

Comment:
I also saw some `project.VersionBundle("PROVIDER").Version` calls in the code. Can you grep for them and replace with `project.BundleVersion()`?

Reply:
Done